server/data/prisonApiClient.ts

```diff
@@ -109,4 +111,18 @@ export default class PrisonApiRestClient implements PrisonApiClient {
       return error
     }
   }
+
+  async getEventsScheduledForToday(bookingId: number): Promise<ScheduledEvent[]> {
+    try {
+      const result = await this.restClient.get({
+        path: `/api/bookings/${bookingId}/events/today`,
+      })
+      return result as ScheduledEvent[]
```

Comment:
If you type the API get call like `restClient.get<ScheduledEvent[]>()` you don't need the `as ScheduledEvent[]` on the return

Reply:
Did this and then also did a big refactor 🎉